### PR TITLE
[VRF-892] Remove noisy log poller warning for VRFv2(+) jobs

### PR DESCRIPTION
**Pull request by vreff**
Removes logic that repeatedly calls the Log Poller's `RegisterFilter` function, which fires off a noisy warning for duplicates. Instead checks the log poller for an existing filter and registers one if not found.

### CODE REVIEW

**Review by leeyikjiun — APPROVED** [review]

**Review by jinhoonbang — DISMISSED** [review]

File: core/services/vrf/v2/listener_v2_log_listener.go [hunk 1]
```diff
@@ -39,36 +40,35 @@ func (lsn *listenerV2) runLogListener(
...
+					lsn.l.Errorw("error registering filter in log poller, retrying",
+						"err", err,
+						"elapsed", time.Since(start))
+					continue
+				}
 			}
 
 			// on startup we want to initialize the last processed block
 			if startingUp {
+				var err2 error
```
Comment (leeyikjiun) [inline core/services/vrf/v2/listener_v2_log_listener.go:66]: I think it should be fine naming it as `err` instead of `err2`?

**Review by jinhoonbang — DISMISSED** [review]

File: core/services/vrf/v2/listener_v2_log_listener.go (hunk 1 above)
Comment (vreff) [reply core/services/vrf/v2/listener_v2_log_listener.go:66]: Yeah, I think you're right that it wouldn't cause a shadowing issue.

**Comment by vreff** [comment]
Sorry for the re-review request here, the PR had an orphaned goroutine in the test so it was causing failures in GH actions. Also, added a git changeset.

**Review by jinhoonbang — DISMISSED** [review]

**Comment by jinhoonbang** [comment]
looks good but there is a merge conflict again. will re approve once that's resolved

**Review by kidambisrinivas — APPROVED** [review]

**Pull request merged**